app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
```

Comment:
The import statement should use double quotes to maintain consistency with the preferred style of quotes.